test/utils/image/manifest.go

```diff
@@ -71,18 +75,20 @@ var (
 	Kitten                   = ImageConfig{e2eRegistry, "kitten", "1.0", true}
 	Liveness                 = ImageConfig{e2eRegistry, "liveness", "1.0", true}
 	LogsGenerator            = ImageConfig{e2eRegistry, "logs-generator", "1.0", true}
-	Mounttest                = ImageConfig{e2eRegistry, "mounttest", "1.0", true}
-	MounttestUser            = ImageConfig{e2eRegistry, "mounttest-user", "1.0", true}
-	Nautilus                 = ImageConfig{e2eRegistry, "nautilus", "1.0", true}
-	Net                      = ImageConfig{e2eRegistry, "net", "1.0", true}
-	Netexec                  = ImageConfig{e2eRegistry, "netexec", "1.0", true}
-	Nettest                  = ImageConfig{e2eRegistry, "nettest", "1.0", true}
-	NginxSlim                = ImageConfig{gcRegistry, "nginx-slim", "0.20", true}
-	NginxSlimNew             = ImageConfig{gcRegistry, "nginx-slim", "0.21", true}
-	Nonewprivs               = ImageConfig{e2eRegistry, "nonewprivs", "1.0", true}
-	NoSnatTest               = ImageConfig{e2eRegistry, "no-snat-test", "1.0", true}
-	NoSnatTestProxy          = ImageConfig{e2eRegistry, "no-snat-test-proxy", "1.0", true}
-	NWayHTTP                 = ImageConfig{e2eRegistry, "n-way-http", "1.0", true}
+	// This image is primarily used for verifying image locality priority scheduling function given its relatively large size
+	MnistTpu        = ImageConfig{e2eRegistry, "mnist-tpu", "1.0", true}
```

Comment:
where exactly is the source code for this image(`mnist-tpu`)? We expect to have the source code for the test images in https://github.com/kubernetes/kubernetes/tree/master/test/images to enable it for multiarch and easy to maintain it.

Reply:
Oh, did not know this place before, thanks for reminding!

@silveryfu Please add a `LargeImage` here by following @mkumatag suggestion. 

And I would also suggest you to make sure the image with virtual size of 3~4GB so cri-containerd would be happy.

Reply:
>if it is not a real size then why don't you mention that number directly in the testcase instead of defining here?

@mkumatag I moved this function out to the test's scope - thank your for the suggestion!

> where exactly is the source code for this image(mnist-tpu)? We expect to have the source code for the test images in https://github.com/kubernetes/kubernetes/tree/master/test/images to enable it for multiarch and easy to maintain it.

Thank you for pointing this out! The current maintained images are relatively small in size so we picked this Mnist-Tpu image hosted, upon my previous check, on e2eRegistry. 

I added TODO for adding a LargeImage and adding its source code to be maintained.